docs/source/executing-operations/requests.mdx

```diff
@@ -1,16 +1,16 @@
 ---
 title: Operation Request Format
-subtitle: Send requests to the Apollo Router over HTTP
-description: Learn how to format and send GraphQL requests to Apollo Router over HTTP. Explore POST and GET  examples and the automatic persisted queries protocol.
+subtitle: Send requests to the GraphOS Router over HTTP
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)